packages/react/src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -97,7 +98,9 @@ class OverflowMenu extends Component {
     /**
      * The ARIA label.
      */
-    ariaLabel: PropTypes.string,
+    ariaLabel: FeatureFlags.enabled('enable-v11-release')
+      ? PropTypes.string.isRequired
+      : PropTypes.string,
```

Comment:
In this component, are we looking for `ariaLabel` or `iconDescription` to be the accessibility label for the component?

Reply:
I figured iconDescription was not necessarily the accessibility label